updatecli/updatecli.d/charts/accountapp.yaml

```diff
@@ -11,8 +11,9 @@ targets:
     name: "Update the chart version for accountapp"
     kind: yaml
```

Comment:
```suggestion
    kind: file
```

Reply:
Because: 
```text
TARGETS

========


updateChartVersion

------------------


**Dry Run enabled**


ERROR: Something went wrong in target "updateChartVersion" :

ERROR: Validation error: the provided manifest configuration had the following validation errors:

Invalid spec for yaml resource: 'key' is empty.
```
On the CI